CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Changes since v2.15
 ### Changes
 - REMS no longer sends the Server: HTTP header to avoid leaking version information. (#2216)
 - Text descriptions of some events in the log were phrased better. The created event also shows the original external id. (#2614)
+- REMS no longer uses the database time when creating rows. Requires migration, but does not change visible behaviour. (#2540)
```

Comment:
Maybe would mention that it uses application server time now consistently instead.